src/main/java/org/opentripplanner/service/vehiclerental/model/VehicleRentalStation.java

```diff
@@ -188,4 +189,22 @@ public Set<RentalFormFactor> formFactors() {
       getAvailablePickupFormFactors(false)
     );
   }
+
+  public List<RentalVehicleTypeCount> getVehicleTypeCounts() {
+    return vehicleRentalTypeMapToList(vehicleTypesAvailable);
+  }
+
+  public List<RentalVehicleTypeCount> getVehicleSpaceCounts() {
+    return vehicleRentalTypeMapToList(vehicleSpacesAvailable);
+  }
```

Comment:
This file is not a good example but we would like to see Javadoc for all public methods.